setup/pi/setup-teslausb

```diff
@@ -27,8 +27,7 @@ function read_setup_variables {
   newnamefor[musicsharename]=MUSIC_SHARE_NAME
   newnamefor[shareuser]=SHARE_USER
   newnamefor[sharepassword]=SHARE_PASSWORD
-  newnamefor[tesla_email]=TESLA_EMAIL
-  newnamefor[tesla_password]=TESLA_PASSWORD
+  newnamefor[tesla_refresh_token]=TESLA_REFRESH_TOKEN
```

Comment:
This section is for providing backwards compatibility with old config files. Since this is a new setup variable, this isn't needed here.